snapcraft/storeapi/errors.py

```diff
@@ -170,3 +170,31 @@ class StoreReviewError(StoreError):
     def __init__(self, result):
         self.fmt = self.__messages[result['code']]
         super().__init__()
+
+
+class StoreReleaseError(StoreError):
+
+    __FMT_NOT_REGISTERED = (
+        'Sorry, try `snapcraft register {snap_name}` before trying to '
+        'release or choose an existing revision.')
```

Comment:
I'm not convinced about this message, mainly because of the second part. I think I would just remove "or choose an existing revision". Or, it could say "or choose an already registered snap name". I'm not sure if that's better though.